src/definitions/npc_configs.rs

```diff
@@ -203,7 +226,26 @@ impl NpcConfig {
                 109 => npc.slow_walk = Some(false),
                 111 => npc.animate_idle = Some(false),
                 113 => npc.shadow = Some(Shadow::deserialize(&mut buffer)),
+                #[cfg(not(feature = "osrs"))]
```

Comment:
```suggestion
                #[cfg(feature = "rs3")]
```

Please add this to the field above as well :)

Reply:
Added, thanks 👍